### PR TITLE
docs: follow-ups from the book re-org

### PR DESCRIPTION
**Pull request by digorithm**
A couple more updates, nits, etc., related to https://github.com/FuelLabs/fuels-rs/issues/365.

### CODE REVIEW

File: docs/src/wallets/test-wallets.md [hunk 1]
```diff
@@ -26,3 +26,23 @@ You can create a test wallet containing multiple assets (including the base asse
 
 - coins: `Vec<(UtxoId, Coin)>` has num_assets * coins_per_assets coins (UTXOs)
 - asset_ids: `Vec<AssetId>` contains the num_assets randomly generated `AssetId`s (always includes the base asset)
+
+## Setting up assets
+
+The Fuel blockchain holds many different assets; you can create your asset with its unique `AssetId` or create random assets for testing purposes.
+
+You can use only one asset to pay for transaction fees and gas: the base asset, whose AssetId is `0x000...`, a 32-byte zeroed value.
```
Comment (vnepveu) [inline docs/src/wallets/test-wallets.md:34]:
```suggestion
You can use only one asset to pay for transaction fees and gas: the base asset, whose AssetId is `0x000...0`, a 32-byte zeroed value.
```

File: docs/src/wallets/test-wallets.md [hunk 2]
```diff
@@ -26,3 +26,23 @@ You can create a test wallet containing multiple assets (including the base asse
 
 - coins: `Vec<(UtxoId, Coin)>` has num_assets * coins_per_assets coins (UTXOs)
 - asset_ids: `Vec<AssetId>` contains the num_assets randomly generated `AssetId`s (always includes the base asset)
+
+## Setting up assets
```
Comment (vnepveu) [inline docs/src/wallets/test-wallets.md:30]: There is a `Setting up a test wallet with multiple assets` section right above, it feels like a duplication. What's more, the ANCHORs overlap. Maybe the above section could be re-worded instead?

**Review by vnepveu — CHANGES_REQUESTED** [review]
Good! Just one question about a duplicate paragraph.

File: docs/src/wallets/test-wallets.md (hunk 2 above)
Comment (digorithm) [reply docs/src/wallets/test-wallets.md:30]: Yeah, I had this concern too. After reading `Setting up a test wallet with multiple assets` with fresh eyes, I noticed it didn't say much about setting up assets and coins, it just made _use_ of it, which is essentially a reference to something that hasn't been "defined" yet (or "talked about") and could be a source of confusion.

I could talk about setting about assets and then say

> For testing purposes, you can configure coins and amounts for assets. You can use setup_multiple_assets_coins as seen above.

And skip the code block, but that felt empty, somehow. So I decided to experiment with _some_ duplication here (I'm not really duplicating it all, just an inner section of the outer anchor), just focusing on the code for setting up multiple assets. 

I think re-wording the above section to include asset stuff would be kinda like giving more responsibilities to a section that's supposed to be only about setting up a test wallet 🤔

Comment (vnepveu) [reply docs/src/wallets/test-wallets.md:30]: Ok with me then!

**Review by vnepveu — APPROVED** [review]
:ship:

**Pull request merged**